config/honeybadger.yml

```diff
@@ -0,0 +1,2 @@
+---
+api_key: <%= ENV['HONEYBADGER_API_KEY'] %>
```

Comment:
You could delete this file entirely now because HONEYBADGER_API_KEY will be picked up automatically from ENV at runtime.